dali/operators/image/crop/bbox_crop.h

```diff
@@ -19,12 +19,24 @@
 #include <vector>
 #include "dali/pipeline/operator/common.h"
 #include "dali/pipeline/operator/operator.h"
+#include "dali/operators/random/rng_base_cpu.h"
 #include "dali/pipeline/util/operator_impl_utils.h"
 
 namespace dali {
 
 template <typename Backend>
-class RandomBBoxCrop : public Operator<Backend> {
+class RandomBBoxCropImplBase : public OpImplBase<Backend> {
+ public:
+  RandomBBoxCropImplBase(const OpSpec *spec, BatchRNG<std::mt19937_64> &rng)
+    : spec_(*spec), rngs_(rng) {}
+
+ protected:
+  const OpSpec &spec_;
+  BatchRNG<std::mt19937_64> &rngs_;
+};
```

Comment:
Do we still need this intermediate class?

Reply:
Gone!